uint64 timeStamp
uint16 fS
uint16 nPots
float32[] data
